easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-6.0-centos6_x86_64.eb

```diff
@@ -0,0 +1,44 @@
+name = 'FreeSurfer'
+version = '6.0.0'
+versionsuffix = '-centos6_x86_64'
+
+homepage = 'http://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferWiki'
+description = """FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
+FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-Linux%(versionsuffix)s-stable-pub-v%(version)s.tar.gz',
+           'runtime2012bLinux.tar.gz']
+source_urls = ['ftp://surfer.nmr.mgh.harvard.edu/pub/dist/%(namelower)s/%(version)s',
+               'http://surfer.nmr.mgh.harvard.edu/fswiki/MatlabRuntime?action=AttachFile&do=get&target=']
```

Comment:
please reformat to one entry per line?

```python
source_urls = [
    ...,
    ...,
]
```

soon we'll have automatic style testing, so it'll be Travis complaining about stuff like this, not me ;)